src/main-client.ts

```diff
@@ -522,526 +522,6 @@ export class MainClient extends BaseRestClient {
-  }
-
-  withdrawAssetsFromManagedSubAccount(
-    params: WithdrawAssetsFromManagedSubAccountParams,
-  ): Promise<MarginTransactionResponse> {
-    return this.postPrivate('sapi/v1/managed-subaccount/withdraw', params);
-  }
-
-  getManagedSubAccountSnapshot(
-    params: ManagedSubAccountSnapshotParams,
-  ): Promise<ManagedSubAccountSnapshot> {
-    return this.getPrivate(
-      'sapi/v1/managed-subaccount/accountSnapshot',
-      params,
-    );
-  }
-
-  getManagedSubAccountTransfersInvestor(
-    params: ManagedSubAccountTransferLogParams,
-  ): Promise<{
-    managerSubTransferHistoryVos: ManagerSubTransferHistoryVos[];
-    count: number;
-  }> {
-    return this.getPrivate(
-      'sapi/v1/managed-subaccount/queryTransLogForInvestor',
-      params,
-    );
-  }
-
-  getManagedSubAccountTransfersParent(
-    params: ManagedSubAccountTransferLogParams,
-  ): Promise<{
-    managerSubTransferHistoryVos: ManagerSubTransferHistoryVos[];
-    count: number;
-  }> {
-    return this.getPrivate(
-      'sapi/v1/managed-subaccount/queryTransLogForTradeParent',
-      params,
-    );
-  }
-
-  getManagedSubAccountFuturesAssets(params: {
-    email: string;
-  }): Promise<ManagedSubAccountFuturesAssetsResponse> {
-    return this.getPrivate(
-      'sapi/v1/managed-subaccount/fetch-future-asset',
-      params,
-    );
-  }
-
-  getManagedSubAccountMarginAssets(params: {
-    email: string;
-  }): Promise<ManagedSubAccountMarginAssetsResponse> {
-    return this.getPrivate('sapi/v1/managed-subaccount/marginAsset', params);
-  }
-
-  getSubAccountAssetsMaster(params: { email: string }): Promise<{
-    balances: SubaccountBalances[];
-  }> {
-    return this.getPrivate('sapi/v4/sub-account/assets', params);
-  }
-
-  getManagedSubAccounts(params: ManagedSubAccountListParams): Promise<{
-    total: number;
-    managerSubUserInfoVoList: ManagerSubUserInfoVo[];
-  }> {
-    return this.getPrivate('sapi/v1/managed-subaccount/info', params);
-  }
-
-  getSubAccountTransactionStatistics(params: {
-    email: string;
-  }): Promise<SubAccountTransactionStatistics> {
-    return this.getPrivate(
-      'sapi/v1/sub-account/transaction-statistics',
-      params,
-    );
-  }
-
-  getManagedSubAccountDepositAddress(
-    params: ManagedSubAccountDepositAddressParams,
-  ): Promise<ManagedSubAccountDepositAddress> {
-    return this.getPrivate(
-      'sapi/v1/managed-subaccount/deposit/address',
-      params,
-    );
-  }
-
-  enableOptionsForSubAccount(params: {
-    email: string;
-  }): Promise<EnableOptionsForSubAccountResponse> {
-    return this.postPrivate('sapi/v1/sub-account/eoptions/enable', params);
-  }
-
-  getManagedSubAccountTransferLog(
-    params: ManagedSubAccountTransferTTLogParams,
-  ): Promise<{
-    managerSubTransferHistoryVos: ManagerSubTransferHistoryVos[];
-    count: number;
-  }> {
-    return this.getPrivate(
-      'sapi/v1/managed-subaccount/query-trans-log',
-      params,
-    );
-  }
-
   /**
```

Comment:
For the broker (link) endpoints, these are in the docs here:
https://developers.binance.com/docs/binance_link/Introduction

Some might be unofficial but usually have a doc. Let me know if there's any you're not sure about that aren't in the docs.